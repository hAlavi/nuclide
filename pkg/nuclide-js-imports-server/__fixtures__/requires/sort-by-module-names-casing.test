const B = require('Abc');
const A = require('AB');
const D = require('CDe');
const C = require('Cd');

const x = [A, B, C, D];
