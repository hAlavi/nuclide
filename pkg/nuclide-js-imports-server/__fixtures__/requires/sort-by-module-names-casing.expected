const A = require('AB');
const B = require('Abc');
const C = require('Cd');
const D = require('CDe');

const x = [A, B, C, D];
